development/tools/jobs/tester/genericcomponentsuite.go

```diff
@@ -94,7 +94,17 @@ func (s GenericComponentSuite) repositoryName() string {
 }
 
 func (s GenericComponentSuite) jobConfigPath() string {
-	return fmt.Sprintf("./../../../../prow/jobs/%s/%s/%s%s.yaml", s.repositoryName(), s.Path, s.componentName(), s.JobsFileSuffix)
+	if strings.Contains(s.Repository, "kyma-project") {
```

Comment:
Maybe we can refactor this to be more bulletproof. Maybe you can detect the `kyma-repository` and `kyma-incubator` repositories and handle them appropriately in other cases just fail.